config.default

```diff
@@ -132,6 +132,15 @@ mpasInterpolationMethod = bilinear
 startYear = 1
 endYear = 9999
 
+[index]
+## options related to producing nino index.
+
+# start and end years for the nino 3.4 analysis.  Using out-of-bounds values
+#   like start_year = 1 and end_year = 9999 will be clipped to the valid range
+#   of years, and is a good way of insuring that all values are used.
+startYear = 2
+endYear = 6
```

Comment:
I talked with @milenaveneziani a bit more about this and I now understand the benefit of the files.  I was not aware you could specify one of those and then if something is missing it is filled from config.default.  Definitely makes sense now.  I'm making the changes now.